packages/twenty-server/src/workspace/workspace-health/services/field-metadata-health.service.ts

```diff
@@ -165,17 +169,28 @@ export class FieldMetadataHealthService {
       });
     }
 
-    if (
-      defaultValue &&
-      columnStructure.columnDefault &&
-      !columnStructure.columnDefault.startsWith(defaultValue)
-    ) {
-      issues.push({
-        type: WorkspaceHealthIssueType.COLUMN_DEFAULT_VALUE_CONFLICT,
-        fieldMetadata,
-        columnStructure,
-        message: `Column ${columnName} default value is not the same as the field metadata default value "${columnStructure.columnDefault}" !== "${defaultValue}"`,
-      });
+    if (defaultValue && columnDefault) {
```

Comment:
I feel like this is a bit difficult to read with nested if here and the method is a bit large. I would either return early in the first if or extract what's inside the if in a dedicated method that would return issues (or nothing), it seems we are doing a similar check in the code below so maybe we could benefit more from a method? What do you think?

Reply:
also columnDefaultValue + metadataDefaultValue would be a bit clearer

Reply:
Actually it's not formatted the same way in both if condition, but I can split the two if here so it's more readable